src/Compilers/VisualBasic/Portable/Scanner/Scanner.vb

```diff
@@ -878,9 +878,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
                             Return
                         End If
 
-                        For i = 0 To triviaList.Count - 1
-                            tList.Add(triviaList(i))
-                        Next
+                        tlist.AddRange(MakeTriviaArray(triviaList))
```

Comment:
this seems to be instantiating an intermediary collection now, just to deconstruct it.  that doesn't seem desirable.

Reply:
The `MakeTriviaArray` is badly named as it doesn't return an array, but a `SyntaxList(Of VisualBasicSyntaxNode)`

Reply:
I realize that :)  But that SyntaxList causes allocations.  That seems highly undesirable in this codepath.